src/include/users/widgets.rb

```diff
@@ -195,10 +195,8 @@ def initialize_users_widgets(include_target)
               VSpacing(0.2),
               "defaults_global",
               "defaultgroup",
-              "groups",
               "shell",
               HBox("home", VBox(Label(""), "browse_home")),
-              HBox("skel", VBox(Label(""), "browse_skel")),
```

Comment:
can you paste to description also screenshot how it looks after that removal? Just to get impression how it looks like.